docs/staking.md

```diff
@@ -0,0 +1,46 @@
+
+:::note
+
+In a Substrate-based network, time is divided in units called **eras**. In [HydraDX Snakenet](/snakenet), *1 Era = 24 hours*.
+
+:::
+
+Participating as a validator requires a certain level of technical knowledge needed for safely setting up and maintaining a validator node. Misbehavior of a validator node can be punished by slashing, leading to an involuntary loss of funds for you and your nominators. If you think you have the needed experience for running a validator node, you can refer to our [validator guide](/node_setup). Otherwise, we highly recommend that you consider becoming a nominator.
+
+### Nominators
+
+Nominators help secure the network by nominating validators to be elected in the active validator set. They do so by staking their HDX tokens with validators of their choice. The nomination process does not require running and maintaining nodes which makes this form of staking more accessible to people lacking a strong technical background. Tokens which are used to nominate validators are *bonded*, meaning that they are frozen and cannot be used for other purposes. Nominators can always withdraw their nomination which will be reflected at the end of the current era. Nominators can also free up their tokens, however this will become only effective after a waiting period of *28 days* following the unbonding transaction.
+
+The choice of validators that you nominate is important for the amount of rewards you are about to receive. You should always do your due diligence before nominating a validator. If your chosen validator fails to live up to the minimum expectations and misbehaves in the network (e.g. too much downtime), it can be punished by slashing which will result in a partial loss of the funds that you have staked with this validator.
+
+The first thing you need to consider when choosing a validator is the *reward commission percentage*. This represents the proportion of the rewards which will be paid out to the validator for providing its services to nominators. Furthermore, you should conduct some investigation into the credibility of validators including checking their identity, as well as historical information such as era points, elected stake, rewards and slashes. At the beginning of Snakenet it might be difficult to find all this information - if you are in doubt about the choice of validators, approach as in Discord and we will share with you our community list of trusted validators.
```

Comment:
I would switch due dilligence as first consideration as it's risky to randomly select validators without it. The commission might be bigger because it costs more for validators with beefier setups, so that is not the best indication how you should select one.